src/app/components/shopping-lists/shopping-lists.component.html

```diff
@@ -14,9 +14,35 @@
   <mat-tab label="Codzienne"></mat-tab>
   <mat-tab label="Niecodzienne"></mat-tab>
 </mat-tab-group>
-<p>
-Hej!<br>
-Sporządź pierwszą<br>
-listę zakupów
-</p>
-<button mat-fab a [routerLink]="'/new-shopping-list'" class="example-button-container plus-button">+</button>
\ No newline at end of file
+<div *ngIf = "lists.length > 0">
+  <mat-accordion>
+    <mat-expansion-panel (opened)="panelOpenState = true"
+    (closed)="panelOpenState = false" *ngFor = "let list of lists">
+      <mat-expansion-panel-header>
+        <mat-panel-title>
+          {{ list.name }}
+        </mat-panel-title>
+      </mat-expansion-panel-header>
+        <div *ngIf = "products.length > 0">
+          <p *ngFor = "let product of products">
```

Comment:
Nie wiem jak tutaj coś zrobić

Reply:
Zrobiłbym tak:
1. <p> zamieniłbym na div'a bo to nie jest paragraf a kontener, w którym będzie narysowany wiersz
2. W tym "nowym" div'ie rysujesz info o produktach -> nie ma tego na mockup'ach więc musisz wynyśleć wygląd, ale generalnie rysujesz normalnie pojedynczy wiersz w html'u a do danych produktu odnośisz się poprzez {{ product.XXX }} gdzie XXX to jest nazwa pola z klasy Product
3. To w zasadzie tyle, bo tutaj jak sądzę ma być tylko podgląd produktów z listy